infra/gcp/terraform/k8s-infra-prow-build/deploy.sh

```diff
@@ -36,17 +36,27 @@ function deploy_cluster_terraform() {
 }
 
 function deploy_cluster_resources() {
+    pushd "${SCRIPT_ROOT}/${cluster}/resources"
```

Comment:
note I opted to push into the resources dir rather than use absolute paths for everything; can rewrite to use absolute paths if you would prefer, but this approach meant less `sed`